penrose-web/src/engine/EngineUtils.ts

```diff
@@ -38,7 +38,20 @@ function mapFloat<T, S>(f: (arg: T) => S, v: IFloatV<T>): IFloatV<S> {
   };
 }
 
-function mapPt<T, S>(f: (arg: T) => S, v: IPtV<T>): IPtV<S> {
+function mapInt<T, S>(
+  f: (arg: T) => S,
+  v: IIntV<T>
+): IFloatV<S> {
+  return {
+    tag: "FloatV",
+    contents: f(v.contents as unknown as T) as S // Hack because we know that Ints only contain `number`s
```

Comment:
Did TS throw an error for this? Surprised that it couldn't figure out `IntV` and `FloatV` have the same types

Reply:
Thanks for pointing this out, I decided to get rid of it as the code currently doesn't aim to do int handling. (In general TS throws an error because `Int` always contains `number`, but the conversion function is general from `T` to `S`, and it can' t unify `Int` with `T`, which makes sense. I don't think here is the right place to handle Int conversion anyway.)

Reply:
Well in this case we should probably change `IIntV` to a non-generic type instead

Reply:
Done in 02977b327e2e89d8ede9c0f5ecef51d71e6ff438